ui/src/components/view/DetailsTab.vue

```diff
@@ -37,7 +37,7 @@
     size="small"
     :dataSource="fetchDetails()">
     <template #renderItem="{item}">
-      <a-list-item v-if="item in dataResource && !customDisplayItems.includes(item)">
+      <a-list-item v-if="item in dataResource && !customDisplayItems.includes(item) || offeringDetails.includes(item)">
```

Comment:
```suggestion
      <<a-list-item v-if="(item in dataResource && !customDisplayItems.includes(item)) || (nestedDetails.includes(item) && dataResource.serviceofferingdetails)">
```
To prevent the header to be listed in offerings that don't have these params